index.js

```diff
@@ -32,6 +32,15 @@ module.exports = () => {
       incoming_webhook_channel: req.headers['bb-incomingwebhookchannel']
     })
 
+    // Add custom config
+    req.slapp.meta.config = Object.keys(req.headers)
```

Comment:
Good point.  I think keeping it as `config` works well as it's consistent with the public api for getting teams (is called `config` there).